### PR TITLE
Added additional attribute to manifest

### PR DESCRIPTION
**Pull request by epananth**
Added  IsReleaseOnlyPackageVersion attribute to the merged manifest

### CODE REVIEW

File: src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/PushMetadataToBuildAssetRegistry.cs [hunk 1]
```diff
@@ -655,7 +657,7 @@ private AssetData GetManifestAsAsset(IImmutableList<AssetData> assets, string lo
                             Branch = GetAzDevBranch(),
                             Commit = GetAzDevCommit(),
                             IsStable = IsStableBuild.ToString(),
-                            PublishingVersion = manifestBuildData.PublishingVersion.ToString()
+                            IsReleaseOnlyPackageVersion = IsReleaseOnlyPackageVersion.ToString()
                         });
```
Comment (epananth) [inline src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/PushMetadataToBuildAssetRegistry.cs:661]: Idk why this has PublishingVersion here :/

**Comment by epananth** [comment]
Need arcade update from '.Net Eng latest' channel, only then we can merge this PR.

**Comment by missymessa** [comment]
> Need arcade update from '.Net Eng latest' channel, only then we can merge this PR.

You can trigger the subscription to pull the latest. I promoted Arcade version 5.0.0-beta.20506.7 this afternoon :)

**Comment by epananth** [comment]
We stopped arcade update in arcade-services like 6 months ago, so testing to see if the update actually breaks anything :(

**Comment by epananth** [comment]
closing this duplicate draft

**Pull request closed without merging**